### ISSUE TITLE
🐛 BUG: `is:global` not being recognised with the vscode extension

### ISSUE THREAD

**Issue by Tc-001** [withastro/language-tools#241]
### Describe the Bug

`<style is:global>` throws a typescript error

### Steps to Reproduce

1. `npm init astro` using any template
2. Replace `index.astro` with:
```astro
<head>
	<style is:global>
	</style>
</head>
```
3. Observe error:
```
Type '{ children: string; "is:global": true; }' is not assignable to type 'HTMLProps<HTMLStyleElement> & AstroDefineVars'.
  Property 'is:global' does not exist on type 'HTMLProps<HTMLStyleElement> & AstroDefineVars'.ts(2322)
```
![image](https://user-images.githubusercontent.com/55956895/161444438-14821305-27d6-4fee-87e7-6867a7e64fce.png)

**Comment by Tc-001** [comment]
https://github.com/withastro/language-tools/blob/780b1a4ea55ba2e3deae895ae00e819de217d5a2/packages/language-server/types/astro-jsx.d.ts#L48

I believe this needs to be changed to `is:global` in order for it to work, but for me even the old syntax does not work, so there may be something deeper to fix
See:
```
Type '{ children: string; global: true; }' is not assignable to type 'HTMLProps<HTMLStyleElement> & AstroDefineVars'.
  Property 'global' does not exist on type 'HTMLProps<HTMLStyleElement> & AstroDefineVars'.ts(2322)
```
![image](https://user-images.githubusercontent.com/55956895/161444669-767e31a7-ecde-4f45-b1d1-9eeed79f963b.png)

**Comment by Princesseuh** [comment]
The history of it is that we have an unreleased fix for `global` (the old syntax) but we already need to change it for the new syntax 😅

Planning to release a fix for this and other 0.26.0 related issues as soon as possible
